src/cloud-element-templates/properties-panel/properties/custom-properties/index.js

```diff
@@ -217,4 +217,9 @@ function getDefaultType(property) {
   }
 }
 
+function isEdited(isEditedEntry, property) {
```

Comment:
I'd suggest to short-circuit properties with `generatedValue` as we don't want to generate a new one per each render.

Reply:
```suggestion
function isEdited(isEditedEntry, property) {

  // generated values are always marked as edited
  if (property.generatedValue) {
    return true;
  }
```